README.md

```diff
@@ -22,19 +20,41 @@ Usage:
 Available Commands:
   dinghy      Run dinghy subcommands
   help        Help about any command
+  version     Prints version information
 
 Flags:
-  -h, --help   help for arm
+  -h, --help              help for arm
+  -l, --loglevel string   log level (default "info")
 
 Use "arm [command] --help" for more information about a command.
 ```
 
+##Mac OS
```

Comment:
Need to include a space between `#` and `M` so the GH MD render doesn't break:
https://s.armory.io/jkulvrB2

Reply:
Changed! Thanks!